watchdog-config/extra-sql-queries.yml

```diff
@@ -75,3 +75,400 @@ users_with_insificient_disk:
+        monitoring_urls ON docs.partner_name=monitoring_urls.partner_name
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - invalid_doc_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+invalid_doc_errors:
+  query: |
+    SELECT
+        docs.partner_name,
+        day::varchar(255) AS reported,
+        code,
+        count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_invalid_doc_errors AS docs
+    LEFT JOIN
+        monitoring_urls ON docs.partner_name=monitoring_urls.partner_name
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - code:
+        usage: "LABEL"
+    - count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+tasks_by_state:
+  query: |
+    SELECT
+        tasks.partner_name,
+        day::varchar(255) AS reported,
+        task,
+        draft,
+        ready,
+        cancelled,
+        completed,
+        failed,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_tasks_by_state AS tasks
+    LEFT JOIN
+        monitoring_urls ON tasks.partner_name=monitoring_urls.partner_name
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - task:
+        usage: "LABEL"
+    - draft:
+        usage: "GAUGE"
+    - ready:
+        usage: "GAUGE"
+    - cancelled:
+        usage: "GAUGE"
+    - completed:
+        usage: "GAUGE"
+    - failed:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+tasks_by_user:
+  query: |
+    SELECT
+        tasks.partner_name,
+        chw,
+        task,
+        docs,
+        cancelled,
+        completed,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_tasks_by_user AS tasks
+    LEFT JOIN
+        monitoring_urls ON tasks.partner_name=monitoring_urls.partner_name
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - chw:
+        usage: "LABEL"
+    - task:
+        usage: "LABEL"
+    - docs:
+        usage: "GAUGE"
+    - cancelled:
+        usage: "GAUGE"
+    - completed:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+form_completions:
+  query: |
+    SELECT
+        forms.partner_name,
+        username,
+        form_name,
+        load_count,
+        complete_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_form_completions AS forms
+    LEFT JOIN
+        monitoring_urls ON forms.partner_name=monitoring_urls.partner_name
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - username:
+        usage: "LABEL"
+    - form_name:
+        usage: "LABEL"
+    - load_count:
+        usage: "GAUGE"
+    - complete_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+purge_logs:
+  query: |
+    SELECT
```

Comment:
this one returns 17k results - just about every single one (or all of them!?!) are for `partner_name` of `itech_aurum_scaleup` - maybe exclude them? Or purge that table and re-populate?

Reply:
Using partition by to limit results to top 5 by partner

Reply:
ah - this is just right! I see we have an inner `SELECT` that returns 17k rows, but the outer `SELECT` culls that down to 5 per partner, just as you said.  Having this expensive query run only every 12 hours and return a small amount of important results is great!